packages/redux/src/contents/actions/factories/fetchCommercePagesFactory.ts

```diff
@@ -29,7 +29,7 @@ import type { Dispatch } from 'redux';
 const fetchCommercePagesFactory =
   (getCommercePages: GetCommercePages) =>
   (
-    query: QueryCommercePages,
+    query: QueryCommercePagesWithSlug,
```

Comment:
This will be a breaking change, can't you calculate a default value for the slug or at least make it optional?